utils/synthetics/getMarketsLiquidityInfo.ts

```diff
@@ -0,0 +1,91 @@
+import { getContractAddress, getSyntheticsReaderContract } from '@/config/constants'
+import { getClient } from '@/lib/client'
+import { getMarketsInfo } from './getMarketsInfo'
+import { getContractPrices } from './getPrices'
+
+const USD_DIVISOR = BigInt(Math.pow(10, 30))
```

Comment:
`BigInt(Math.pow(10, 30))` returns `1000000000000000019884624838656n` 

i think it's better to use `10n ** 30n` instead to make it more accurate

Reply:
Fixed it up now, thanks!